### ISSUE TITLE
Thanks, and a question

### ISSUE THREAD

**Issue by bland328** [AlexanderWillner/runMacOSinVirtualBox#13]
_tl;dr: Does 'macOS-MojaveClover.iso' need to stick around as the key to making this all work?_

Thanks very much for creating this, Alexander. It is slick and effective!

Since I'm new to virtualizing macOS, I have a two-part question for you:

Do I understand correctly that the virtual 'macOS-MojaveClover.iso' disc is to stick around permanently as a boot volume, as the key to your VBox/APFS workaround?

Or can I, in the end, inject the Clover bootloader into the EFI partition of the virtual SSD, and boot without the 'macOS-MojaveClover.iso'?

Right now, if I eject the iso and attempt to reboot, I'm dropped into the EFI shell, with only an FS0: that contains very little.

Thank you very much for entertaining my question--I imagine it is very naive ;)

**Comment by bland328** [comment]
Disregard! I gave up too quickly. And instead of deleting my naive question, I'll answer it for the benefit of others.

I booted Mojave in VBox, mounted the EFI partition, copied the CLOVER and BOOT folders from the virtual optical disc into the EFI folder alongside the APPLE folder, ejected the optical, and successfully rebooted.

It all makes so much sense...once it all makes so much sense ;)

Thanks again for creating this wonderful utility, Alexander.

**Comment by AlexanderWillner** [comment]
Interesting. Would be a nice addition to offer a script to automate this step...

**Comment by bland328** [comment]
Do you say "interesting" because it is interesting that I misunderstood the goal in the first place?

Or do you say it because I came up with an interesting approach that wasn't your intent? :)

**Comment by AlexanderWillner** [comment]
Interesting that there is a way to get rid of the clover iso ;)

**Comment by SFrost007** [comment]
For anyone who stumbles upon this in the future (including possibly myself!), the exact commands I ran to replicate @bland328's steps above were:

```
sudo mkdir /Volumes/EFI
sudo mount -t msdos /dev/disk0s1 /Volumes/EFI
sudo cp -r /Volumes/NO\ NAME/EFI/* /Volumes/EFI/EFI
sudo diskutil unmount /dev/disk0s1
```

Then ejected the NO NAME drive from the desktop icon's context menu.

**Comment by AlexanderWillner** [comment]
Nice. Thanks.

**Comment by bland328** [comment]
Yes, thanks! 😃

**Comment by nlsrchtr** [comment]
This does not work for me. How can I debug this behaviour? Can you give me any guidance?

**Comment by AlexanderWillner** [comment]
Does my latest commit help?
